README.md

```diff
@@ -1 +1,204 @@
-# crypto-metrics-webapp
\ No newline at end of file
+# crypto-metrics-webapp
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [crypto-metrics-webapp] <a name="about-project"></a>
+
+> Space travelers hub is a react based website where a person can book rockets and book a mission or even leave rocket and mission. it's developed using different libraries like React, Redux,thunk,bootstrap etc. we used hooks to handle state.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Following technologies have been used in this project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.org/">React JS</a></li>
+    <li><a href="https://www.jsx.com/">JSX</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/current-work.en.html">CSS</a></li>
+    <li><a href="https://www.ecma-international.org/publications-and-standards/standards/ecma-262/">JavaScript5</a></li>
+    <li><a href="https://webpack.js.org/">Webpack Framework</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://pages.github.com/">Github Pages</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> This project is a React Single Page App.
+> You can book missions and rockets.
+> You can leave missions and reserved rockets.
+> This project is using different react libraries library.
+> This project is using React Router.
+> This project is using best practices with React and JavaScript.
+> This project follows the gitflow convention.
+> This is using eslint and stylelint to check errors.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://crypto-yjtf.onrender.com)
+- [demo video](https://www.loom.com/share/cbb1a90bbbd54986878eb2e2867f7e71)
```

Comment:
- [ ] OPTIONAL
Good job with project presentation, however, you missed some steps in your video presentation. Your video presentation only hinted on the app features of your app but failed to highlight any portion of your code. Kindly ensure that the video contains a highlight about a piece of the code that you find interesting or that you were proud of making.